resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/config.scala

```diff
@@ -126,6 +126,8 @@ package object config extends Logging {
       .stringConf
       .createOptional
 
+  private[spark] val KUBERNETES_DRIVER_ENV_KEY = "spark.driverEnv."
```

Comment:
Yeah. Driver configuration key is different but executor is the same across cluster manager.